hadoop-common-project/hadoop-nfs/src/main/java/org/apache/hadoop/portmap/RpcProgramPortmap.java

```diff
@@ -208,4 +208,8 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable t) {
     LOG.warn("Encountered ", t);
     ctx.channel().close();
   }
+
+  public ConcurrentHashMap<String, PortmapMapping> getMap() {
+    return map;
+  }
```

Comment:
This method can be package-private and we should use more abstract return type (in this case, `Map`).

Reply:
I will fix it.